eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -105,10 +105,14 @@ jobs:
     - ${{ if in(parameters.platform, 'Android_arm', 'Android_arm64') }}:
       - Windows.10.Amd64.Android.Open
 
-    # iOS x64/x86
-    - ${{ if in(parameters.platform, 'iOSSimulator_x64', 'iOSSimulator_x86') }}:
+    # iOS simulator x64/x86
+    - ${{ if in(parameters.platform, 'iOSSimulator_x64', 'iOSSimulator_x86', 'tvOSSimulator_x64') }}:
       - OSX.1015.Amd64.Open
 
+    # iOS devices
+    - ${{ if in(parameters.platform, 'iOS_arm64', 'tvOS_arm64') }}:
+      - OSX.1015.Amd64.Iphone.Open
```

Comment:
`tvOS_arm64` should probably go to `OSX.1015.Amd64.AppleTV.Open`

Reply:
Yup.

Reply:
Could also be turned to `eq` instead of `in`